src/commands/remoteDebug/getRemoteDebugLanguage.ts

```diff
@@ -50,8 +50,14 @@ export function getRemoteDebugLanguage(siteConfig: SiteConfigResource, context:
 
 // Remote debugging is currently only supported for Node.js >= 8.11
 function isNodeVersionSupported(nodeVersion: string): boolean {
-    // the portal's new default node runtime is LTS
-    if (nodeVersion.toLocaleLowerCase() === 'lts') {
+    // the portal's Node runtimes includes several LTS versions i.e. (lts, 8-lts, 12-lts)
+    if (/lts/i.test(nodeVersion)) {
+        const ltsVersionNumber: number = Number(nodeVersion.split('-')[0]);
+        // if there is no version number, ltsVersionNumber will equal 'lts' which isNan
+        if (ltsVersionNumber < 8 && !isNaN(ltsVersionNumber)) {
```

Comment:
Shouldn't this be `||`?

Reply:
Nah, it needs to be a number and less than 8.  If it isn't a number, then it's LTS, and that means we want it to skip this condition and return true.